dir.traversal.targets

```diff
@@ -86,6 +86,32 @@
              ContinueOnError="ErrorAndContinue" />
   </Target>
 
+  <Target Name="ProducesPackageId"
+          Returns="@(PackageIds)" 
```

Comment:
@weshaggard , this is scoped to the package .builds files.  You're saying that even that is too much?

Yes, the spec shows package versions, and I expect to wire that up once the API for this is in place.  My goal for the first pass was to get us on par with Roslyn and then focus on doing the same for Consumes.  It seems that the Consumes work will be dependent on the versioning work you're working on, @dagood , and the source of truth which gets checked in.  I haven't started seriously thinking about Consumes yet though.